### ISSUE TITLE
After update to 1.1.x the device-nodes have to be rebuilt

### ISSUE THREAD

**Issue by tt2g89** [mabunixda/node-red-contrib-alexa-home#46]
**Describe the bug**
Updated from 1.0.4 to 1.1.x, last one was 1.1.2. I have to rebuild every node, because the hub seems to have lost the device ids of the device-nodes
**To Reproduce**
install 1.0.4, update to 1.1.2. Try to turn on a light.

**Expected behavior**
Lights switch on or off

**Desktop (please complete the following information):**
 - Alexa Devices generation: Echo Dot gen2, gen3, fire hd 8, Echo gen 1
 - node version 1.1.2
 - is docker used? No

**Comment by mabunixda** [comment]
@tt2g89 what do you mean with rebuilding nodes?

**Comment by tt2g89** [comment]
@mabunixda I had to copy and paste every device node, i.e. Treppenhausbeleuchtung and delete the old ones and link the nodes to the hub and to the loxone-device, which has to be switched and then had to delete the devices in alexa and research for them in order to get it to work again.
I debugged the hub and the device and found out, that before doing so, the hub got the command, but the device-node didn't. Afterwards both (hub and device) sent out a debug-information.
